alpha-build-machinery/make/examples/multiple-binaries/Makefile.test.log

```diff
@@ -1,8 +1,21 @@
 make -C . build
-go build  github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/cmd/oc
-go build  github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/cmd/openshift
+fatal: No names found, cannot describe anything.
+fatal: No names found, cannot describe anything.
+go build -ldflags "-s -w -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.versionFromGit="<redacted_for_diff>" -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.commitFromGit="<redacted_for_diff>" -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.gitTreeState="<redacted_for_diff>" -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.buildDate="<redacted_for_diff>" " github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/cmd/oc
+go build -ldflags "-s -w -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.versionFromGit="<redacted_for_diff>" -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.commitFromGit="<redacted_for_diff>" -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.gitTreeState="<redacted_for_diff>" -X github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/pkg/version.buildDate="<redacted_for_diff>" " github.com/openshift/library-go/alpha-build-machinery/make/examples/multiple-binaries/cmd/openshift
 [[ -f ./openshift ]]
 [[ -f ./oc ]]
+# test version is set correctly when linking
+#   majorFromGit, minorFromGit are deprecated upstream and set to empty value
+#   we avoid comparing time to avoid flakes
+diff <( ./oc | sed '$d' ) <( \
+	echo ""; \
+	echo ""; \
+	git rev-parse --short "HEAD^{commit}" 2>/dev/null; \
+	git describe --long --tags --abbrev=7 --match 'v[0-9]*' || echo 'v0.0.0-unknown'; \
+	(git diff --quiet && echo 'clean') || echo 'dirty'; \
```

Comment:
`x && y || z` will do what you want.

Reply:
parenthesis looked better to me, but why not